src/settings-ui/Settings.UI/Views/ShellPage.xaml

```diff
@@ -145,6 +146,10 @@
                         x:Uid="Feedback_NavViewItem"
                         Icon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xED15;}"
                         Tapped="FeedbackItem_Tapped" />
+                    <NavigationViewItem
```

Comment:
@crutkas @ethan-fang-MS Not a fan of this as we're trying to declutter the NavView. I also don't have any other suggestions - any ideas from your side?

Reply:
This would be an ideal use of a File ribbon menu, however that doesn't exist unfortunately. I am open to any other ideas, but this was the best I could come up with (beyond placing the exit option randomly on one specific page, but that feels awkward and not easy to find).